airflow/www/views.py

```diff
@@ -545,7 +546,7 @@ def dag_stats(self, session=None):
         dr = models.DagRun
 
         allowed_dag_ids = current_app.appbuilder.sm.get_accessible_dag_ids(g.user)
-        if 'all_dags' in allowed_dag_ids:
+        if permissions.RESOURCE_ALL_DAGS in allowed_dag_ids:
```

Comment:
```suggestion
        if permissions.RESOURCE_ALL_DAGS in allowed_dag_ids:
```

This won't ever be true anymore will it, cos `get_accessible_dag_ids` already handles this an filters it to return all dag ids. (This is probably a separate PR, and this change is fine here)

Reply:
Agreed, and agreed.